src/pyfaf/problemtypes/kerneloops.py

```diff
@@ -770,4 +770,7 @@ def check_btpath_match(self, ureport, parser):
 
     def find_crash_function(self, db_backtrace):
         satyr_koops = self._db_backtrace_to_satyr(db_backtrace)
-        return satyr_koops.frames[0].function_name
+        if satyr_koops.frames
```

Comment:
/me facepalms.

Reply:
Guess I should bring my magnifying glass next time.

Reply:
#metoo